### DISCUSSION TITLE
Model Parametres not showing in some endpoints

### DISCUSSION THREAD

**Discussion by rodrgomes** [danny-avila/LibreChat#4634] [Q&A] [answered]
### What happened?

In old versions, I could use model parameters in all endpoints, but now it dont appear in all

### Steps to Reproduce

It shows in Google

![image](https://github.com/user-attachments/assets/fc2cd96c-16f4-4032-a762-9902631495e5)

If I change the endpoint it not show. Like in Anthropic or Openrouter
![image](https://github.com/user-attachments/assets/2438e36a-9c3f-46be-87f6-a4a12b235d40)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment, marked as answer]
For most endpoints, besides Google, they have moved to the right-hand side panel:

![image](https://github.com/user-attachments/assets/27506209-3f2a-4df8-8d54-09cc3fcb2257)

**Comment by rodrgomes** [comment]
Good to know... I always used it the old way, and since Google still had it, I thought it was a bug... thanks
